spec/stub/formula.rb

```diff
@@ -3,10 +3,13 @@
 require "ostruct"
 require "pathname"
 
+HOMEBREW_TAP_NAME_REGEX = %r{\A(?<tap>(?:[^/]+)/(?:[^/]+))/(?:[\w+\-.@]+)\Z}
```

Comment:
Where is this from?

Reply:
It's a combination of [these two patterns](https://github.com/Homebrew/brew/blob/4.4.4/Library/Homebrew/tap_constants.rb#L4-L10). Is there a better way to do this considering it's only for testing?

Reply:
@tt just an inline Ruby comment clarifying that would be handy!